nodes/nomos-executor/executor/src/api/handlers.rs

```diff
@@ -46,7 +46,8 @@ where
             MempoolAdapter = MempoolAdapter,
             Metadata = Metadata,
         > + Send
-        + Sync,
+        + Sync
+        + 'static,
```

Comment:
Why as this added?

Reply:
The Msg started to be generic over backend. I can try to remove this as well, but the opaque service handled demanded the implementation of backend with enough lifetime